docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx

```diff
@@ -173,4 +156,22 @@ Increasing the values of the farmer parameters could increase the plotting speed
 ```
 --out-connections
 --pending-out-connections
-```
\ No newline at end of file
+```
+
+
+### NUMA Support
+
+:::note
+NUMA support will benefit farmers with large, powerful CPUs and lots of RAM available. The required RAM amount depends on the processor and the number of threads.  
+:::
+
+Previously plotting/replotting thread pools were created for each farm separately even though only a configured number of them can be used at a time (by default just one).
+With the introduction of NUMA support, the farmer application has a thread pool manager that will create a necessary number of thread pools that will be allocated to currently plotting/replotting farms. 
+When a thread pool is created, it is assigned to a set of CPU cores and will only be able to use those cores. Pinning doesn’t pin threads to cores 1:1, instead, OS is free to move threads between cores, but only within CPU cores allocated for thread pool. This will ensure plotting for a particular sector only happens on a particular CPU/NUMA node.
```

Comment:
```suggestion
When a thread pool is created, it is assigned to a set of CPU cores and will only be able to use those cores. Pinning doesn’t pin threads to cores 1:1, instead, the OS is free to move threads between cores, but only within CPU cores allocated for the thread pool. This will ensure plotting for a particular sector only happens on a particular CPU/NUMA node.
```